src/tpm2-tss-engine-tcti.c

```diff
@@ -68,7 +73,7 @@ tcti_expand_dlname(const char *shortname)
 static TSS2_RC
 tcti_dlopen(const char *dl_path, dl_handle_t *dlhandle_p)
 {
-    TSS2_RC r;
+    TSS2_RC r = TSS2_RC_SUCCESS;
```

Comment:
I don't like unnecessary initialization of variables. I prefer to have my compiler/analyzer tell me that I missed a case, which is not the case here. So please revert this line.